### PR TITLE
apt: use post_config_hook

### PR DESCRIPTION
**Pull request by lasers**
We make `post_config_hook` for `apt`. Performance tweak.

We prevent this module from complaining too much. `[Errno 2] No such file or directory: 'apt'`.

### CODE REVIEW

File: py3status/modules/apt_updates.py [hunk 1]
```diff
@@ -28,13 +28,18 @@
 import sys
 
 LINE_SEPARATOR = "\\n" if sys.version_info > (3, 0) else "\n"
+STRING_NOT_INSTALLED = "isn't installed"
```
Comment (tobes) [inline py3status/modules/apt_updates.py]: `not installed' might be easier to read

Comment (lasers) [reply py3status/modules/apt_updates.py]: Ya. I agree. Does not look decent anymore in the long run. I plan to ask you about replacing all of them too (in the PR where I added bunch of empty docstrings). For now, just use `isn't installed` and we'll change them all.

Comment (tobes) [reply py3status/modules/apt_updates.py]: I'd prefer to just do it once.

This is why just having a few PRs in the queue is less work for you as well as for me/@ultrabug. So I'm going to wait on them to be correct.

Comment (lasers) [reply py3status/modules/apt_updates.py]: K. Fixed this one.

**Comment by tobes** [comment]
Nice improvement for users 👍

**Pull request merged**